CHANGELOG.md

```diff
@@ -9,6 +9,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased] - ReleaseDate
 
+## [0.7.1] - 2023-07-12
+
+### Added
+
+- Added `FromHex` implementation for MrEnclave and MrSigner
+- Added `From<quote3<&[u8]>> for Quote<Vec<u8>>`
```

Comment:
Especially since we were talking past each other initially should use the correct type names

```suggestion
- Added `From<Quote3<&[u8]>> for Quote3<Vec<u8>>`
```

Reply:
My bad, I must've missed the Shift key